security_service/src/main/java/com/eai/securityservice/model/OtpStatus.java

```diff
@@ -1,4 +1,4 @@
-package com.eai.otpservice.model;
+package com.eai.securityservice.model;
 
 public enum OtpStatus {
     PENDING,
```

Comment:
Create a custom package named "common.enums" and place all enums within this package.